package.json

```diff
@@ -45,6 +45,7 @@
         "rollup": "^4.18.0",
         "rollup-plugin-add-shebang": "^0.3.1",
         "rollup-plugin-preserve-shebangs": "^0.2.0",
-        "typescript": "^5.4.5"
+        "typescript": "^5.4.5",
+        "typescript-eslint": "^7.11.0"
```

Comment:
you might want to use the alpha version of v8. version 7 isn't officially supported with eslint 9. and v8 has a different set of recommended rules, so if we're starting fresh it might be better to target those. you try could removing the ignored rules here and see if they're already removed from the recommended set in v8

Reply:
Good hint. Also tested the new rules, but the two rules bothering us are still present in the alpha.